peps/pep-0710.rst

```diff
@@ -140,6 +140,11 @@ Additionally, the user:password section of the URL MAY be a well-known,
 non-security sensitive string. A typical example is ``git`` in the case of an
 URL such as ``ssh://git@gitlab.com``.
 
+The value of the ``index_url`` key MUST be a base URL of a Python package index
```

Comment:
I believe we can use :term:`Package Index` here instead of "Python package index"?

Reply:
Adjusted, thanks! 👍🏻